db/models/seeders.js

```diff
@@ -0,0 +1,165 @@
+const { query } = require('../index');
+
+module.exports = {
+  seedStudents: (students) => {
+    return new Promise((resolve, reject) => {
```

Comment:
I see.  Promise here.  Async/Await elsewhere.

We should make it consistent.  We can decide that in context A we use A/A and in context B we use promises.  Let's definitely follow-up on your suggestion we look at defining a standard.